server/index.ts

```diff
@@ -51,13 +51,13 @@ app.prepare()
             ctx.body = JSON.stringify({ uptime: process.uptime() });
         });
 
-        router.get('*', async (ctx: ParameterizedContext<Logger>) => {
+        router.get(/.*/, async (ctx: ParameterizedContext<Logger>) => {
```

Comment:
This implementation in version 8 is gone, so we have to specify it explicitly. 😓 
https://github.com/koajs/router/blob/8.x/lib/route.js#L30

Reply:
It can not be helped ;) 👍

Reply:
I think that it would be nice if a regular expression could help convey the intent of the filter.